mobile/src/defaultViewComponents/defaultViewTable.js

```diff
@@ -67,7 +67,7 @@ export default class DefaultViewTable extends Component {
   }
 
   verifyData = (bool) => {
-    let items = Object.values(this.props.items)
+    const items = Object.values(this.props.items) || []
```

Comment:
Object.values always returns an array

Reply:
Even in cases this.props.items is undefined?

Reply:
`Object.values(undefined)` throws an error